src/util/records.py

```diff
@@ -794,6 +804,34 @@ def rename_xmlid(cr, old, new, noupdate=None, on_collision="fail"):
                 # Don't change the view keys unconditionally to avoid changing unrelated views.
                 cr.execute("UPDATE ir_ui_view SET key = %s WHERE key = %s", [new, old])
 
+                # Adapting t-call and t-name references in views
+                search_pattern = r"""\yt-(call|name)=(["']){}\2""".format(old)
```

Comment:
Since we are going to pass this as a PG regexp, escape it first.
```suggestion
                search_pattern = r"""\yt-(call|name)=(["']){}\2""".format(re.escape(old))
```